ledger/acctupdates.go

```diff
@@ -836,7 +836,12 @@ func (aul *accountUpdatesLedgerEvaluator) GenesisProto() config.ConsensusParams
 
 // VotersForStateProof returns the top online accounts at round rnd.
 func (aul *accountUpdatesLedgerEvaluator) VotersForStateProof(rnd basics.Round) (voters *ledgercore.VotersForRound, err error) {
-	return aul.ao.voters.getVoters(rnd)
+	return aul.ao.voters.VotersForStateProof(rnd)
```

Comment:
note: we didn't fully resolve this discussion. So moving it to here
https://github.com/algorand/go-algorand/pull/4887/files#r1051934764